backend/src/services/mempool_monitor.py

```
@@ -1,224 +1,322 @@
+            tx_resp = await self.solana_client.get_transaction(
+                sig,
+                encoding="jsonParsed",
+                max_supported_transaction_version=0
             )
-            logger.info(f"Subscribed to logs with ID: {subscription_id}")
+            if tx_resp and tx_resp.value:
+                sol_amount = await self._get_sol_transfer_amount(tx_resp.value.transaction)
+                if sol_amount < 0.1:
+                    logger.debug(f"Ignoring transaction {signature} with small SOL transfer: {sol_amount} SOL")
+                    return
+        except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)